### DISCUSSION TITLE
Arabic Language support

### DISCUSSION THREAD

**Discussion by abobakeralshahari** [meilisearch/product#690] [Feedback & Feature Proposal]
Does it support the Arabic language ?

**Comment by ManyTheFish** [comment]
Hello @abobakerMohsan.
Yes, the Language is officially supported, I put below the dedicated discussion of the Arabic Language support:
https://github.com/meilisearch/product/discussions/139

If you see any relevancy issues when trying Meilisearch, don't hesitate to post a comment directly in the discussion, someone from the community or from Meilisearch could help you!

See you!

I am closing this discussion in favor of the [official one](https://github.com/meilisearch/product/discussions/139).
